src/Window.js

```diff
@@ -102,6 +102,7 @@ const {
 GlobalContext.id = id;
 GlobalContext.args = args;
 GlobalContext.version = version;
+GlobalContext.userAgent = `Mozilla/5.0 (OS) AppleWebKit/999.0 (KHTML, like Gecko) Chrome/999.0.0.0 Safari/999.0 Exokit/${GlobalContext.version}`;
```

Comment:
We shouldn't add anything new to `GlobalContext` unless there is no other way, Ideally, `GlobalContext` is deleted entirely.

Reply:
Can `fetch` pull from the global `navigator` instead?

Reply:
Any suggestions? The only alternative I can think of is to move this line into `src/fetch.js`. But then it will still depend on `GlobalContext.version`.

Reply:
It could be a method on `window-fetch` that sets the interception user agent.

Reply:
Okay, I've updated the code. `src/fetch.js` now pulls from `Navigator.userAgent`, which is a `static get userAgent() { .... }` property.

Reply:
Alternatively, the `navigator` should be a global in all cases, where the `userAgent` can be accessed.

Reply:
That makes sense, but I don't think it needs to be a static. We could just access the `navigator` instance without adding any new data shapes.

Reply:
The `navigator` instance is per-window, and there can be multiple windows. As far as I know, there isn't a concept of "the global `window` value", so `src/fetch.js` can't really call `window.navigator.userAgent` unless the user passes in `window`, which would complicate the interface.

I think exokit wanted a constant, un-configurable User-Agent string, if I remember our conversation correctly from a few months ago when I proposed a configurable User-Agent string. If so, then it seems to make sense for it to be either attached to `GlobalContext` or a static getter on `Navigator`.

Reply:
> As far as I know, there isn't a concept of "the global window value"

There is, in the specs. For our purposes that is simply `global`, as set up by `node`. I think this should work for all of the worker cases, as well as `window`.